docker/Dockerfile

```diff
@@ -69,22 +69,22 @@ RUN mkdir -p /build/vtk-m && \
   make -j4
 
 # Build stempy
-COPY . /source/stempy
-
-RUN mkdir -p /build/stempy && \
-  cd /build/stempy && \
+RUN cd /source                                                                              && \
```

Comment:
This is going to break building within the repo, can not keep the COPY?

Reply:
Seems GHA checkout defaults to not recursively grabbing submodules (https://github.com/actions/checkout). Latest commit should fix.